Mathlib/Data/Fintype/Basic.lean

```diff
@@ -3,7 +3,6 @@ Copyright (c) 2017 Mario Carneiro. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mario Carneiro
 -/
-import Mathlib.Algebra.Ring.Hom.Defs -- FIXME: This import is bogus
```

Comment:
Note that `Algebra.Ring.Hom.Defs` is *still* imported in `Data.Fintype.Basic`. You're just hiding the problem.